reframe/core/modules.py

```diff
@@ -8,6 +8,7 @@
 
 import reframe.core.fields as fields
 import reframe.utility.os as os_ext
+from collections import OrderedDict
```

Comment:
Please have a look in the [coding style](https://github.com/eth-cscs/reframe/wiki/Coding-Style-Guide#imports) regarding the import statements.